src/features/profile/components/Header/NonBeneficiaryHeader/NonBeneficiaryHeader.native.test.tsx

```diff
@@ -51,149 +64,168 @@ describe('<NonBeneficiaryHeader/>', () => {
     beforeEach(() => {
       mockNextSubscriptionStep = mockStep
     })
-    it('should render the right banner for 18 years old users, call analytics and navigate to nextBeneficiaryValidationStep', async () => {
-      const setError = jest.fn()
-      const { nextBeneficiaryValidationStepNavConfig } =
-        useBeneficiaryValidationNavigation(setError)
+    it('should render the activation banner for 18 years old users', async () => {
+      server.use(
+        rest.get<BannerResponse>(env.API_BASE_URL + '/native/v1/banner', (_req, res, ctx) =>
+          res(
+            ctx.status(200),
+            ctx.json({
+              banner: {
+                name: BannerName.activation_banner,
+                text: 'à dépenser sur l’application',
+                title: 'Débloque tes 1000\u00a0€',
+              },
+            })
+          )
+        )
+      )
 
       render(
-        <NonBeneficiaryHeader
-          eligibilityStartDatetime="2021-03-30T00:00Z"
-          eligibilityEndDatetime="2022-02-30T00:00Z"
-        />
+        reactQueryProviderHOC(
+          <NonBeneficiaryHeader
+            eligibilityStartDatetime="2021-03-30T00:00Z"
+            eligibilityEndDatetime="2022-02-30T00:00Z"
+          />
+        )
       )
 
-      const banner = screen.getByTestId('eligibility-banner')
-      fireEvent.press(banner)
+      expect(screen.queryByTestId('eligibility-banner-container')).toBeTruthy()
 
-      await waitFor(() => {
-        expect(mockedNavigate).toHaveBeenCalledWith(
-          nextBeneficiaryValidationStepNavConfig?.screen,
-          nextBeneficiaryValidationStepNavConfig?.params
-        )
-      })
+      expect(await screen.findByText('Débloque tes 1000\u00a0€')).toBeTruthy()
+      expect(screen.queryByTestId('BicolorUnlock')).toBeTruthy()
+      expect(screen.getByText('à dépenser sur l’application')).toBeTruthy()
     })
 
-    it('should render the right banner for 18 years old users if user has not completed identity check', () => {
-      render(
-        <NonBeneficiaryHeader
-          eligibilityStartDatetime="2021-03-30T00:00Z"
-          eligibilityEndDatetime="2022-02-30T00:00Z"
-        />
+    it('should render the activation banner for transition 17 to 18 years old users', async () => {
```

Comment:
Pareil le nom du test porte à confusion, en général on aime bien nommer les tests "**should** do something **when** blabla"